### PR TITLE
[Fix][PTP] Fixed downloading torrent files for PTP

### PR DESCRIPTION
**Pull request by seeyabye**
Potential fix for https://github.com/Jackett/Jackett/issues/8927

@RoloSoze would appreciate it if you could test this beforehand.

### CODE REVIEW

**Comment by RoloSoze** [comment]
> 
> 
> Potential fix for #8927
> 
> @RoloSoze would appreciate it if you could test this beforehand.

I will test the fix once my IP ban is lifted; I still cannot access PTP. It'll take a few hours. Thank you for your work, @seeyabye!

File: src/Jackett.Common/Indexers/PassThePopcorn.cs [hunk 1]
```diff
@@ -253,5 +258,18 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
...
+
+        public override async Task<byte[]> Download(Uri link)
+        {
+            var authHeaders = new Dictionary<string, string>()
+            {
+                { "ApiUser", configData.User.Value },
+                { "ApiKey", configData.Key.Value }
+            };
+
+            var response = await base.RequestBytesWithCookies(link.ToString(), headers: authHeaders);
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/PassThePopcorn.cs:270]: If the headers are not required, remove this code. If the previous code works for you, the issue is in @RoloSoze configuration and this PR is not required at all.

Comment (seeyabye) [reply src/Jackett.Common/Indexers/PassThePopcorn.cs:270]: This part is probably unnecessary. I plan to undo these changes.

**Comment by seeyabye** [comment]
@RoloSoze if you can test this updated PR, it would be appreciated.

**Comment by ngosang** [comment]
@seeyabye please, rebase this PR.
@RoloSoze could you test it?

**Comment by RoloSoze** [comment]
@ngosang @seeyabye Thank you guys, I'll get the chance to test it tonight. Been busy lately. Apologies for the delay.

**Comment by RoloSoze** [comment]
@seeyabye @ngosang YESSSS! This PR fixes the PTP issue! I can now download .torrent files straight from Jackett! Please merge it, and thank you for your hard work, @seeyabye! Apologies once again for the delay!

**Pull request merged**